ynput/ayon-ftrack
# AYON to ftrack sync: Sync comments

## Changelog Description
Sync comments from AYON to ftrack and from ftrack to AYON.

## Additional review information
Both syncs are ONLY live, so if transmitter is not running or leecher is not running we might loose some commets meanwhile. Which I don't know how to "fix". I considr this as first version that is working in "base".

To avoid clashes I also had to remove Integrate note publish plugin which would cause doubled comments.

## Testing notes:
If running service tools, make sure to run `install` command first.
Make sure to use last server version.

1. Run all services.
2. Create comment on AYON/ftrack, validate those are propagated to ftrac/AYON.
3. Change/Remove them -> validate it is propagated to the other side.
4. Comments created during publishing should be pushed to both even if IntegrateNote plugins is not available anymore.

Resolves https://github.com/ynput/ayon-ftrack/issues/170
